### PR TITLE
Crn 1026 integrate the new component for past events in the dashboard

### PR DESCRIPTION
**Pull request by lctrt**

### CODE REVIEW

File: packages/react-components/src/templates/__tests__/DashboardPageBody.test.tsx [hunk 1]
```diff
@@ -72,6 +74,31 @@ it('hides add links to your work space section when user is not a member of a te
   expect(screen.queryByText(/Add important links/i)).toBeNull();
 });
 
+describe('the past events card', () => {
+  const events = [
+    createEventResponse({}, 0),
```
Comment (oratiu) [inline packages/react-components/src/templates/__tests__/DashboardPageBody.test.tsx:79]: Can't you pass createListEventResponse(3) there ?

Comment (lctrt) [reply packages/react-components/src/templates/__tests__/DashboardPageBody.test.tsx:79]: didn't know it existed, will do that!

File: apps/crn-frontend/src/dashboard/Body.tsx [hunk 2]
```diff
@@ -1,5 +1,25 @@
...
+  const { items: events } = useEvents(
+    getEventListOptions(date, {
+      past: true,
+      pageSize: 3,
+      currentPage: 1,
+    }),
+    user,
+  );
+
+  return <DashboardPageBody events={events} {...props} />;
```
Comment (oratiu) [inline apps/crn-frontend/src/dashboard/Body.tsx:22]: I think we should rename it to pastEvents to be more verbose

**Review by oratiu — APPROVED** [review]
Looks good.

**Pull request merged**